include/bootstrap.php

```diff
@@ -3,6 +3,14 @@
 // Used for performance calculations
 $dStartTime = microtime(true);
 
+// set locale
+$language_code = 'ru_RU.utf8';
+putenv("LANG=$language_code");
+setlocale(LC_ALL, $language_code);
+$domain = 'pool_ru';
```

Comment:
If this refers to the file, maybe this would be better to be derived from the language code (`ru_RU.utf8` we could use the two letters before `_RU.utf8`?).